src/control/taskControl.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-undef */
+import { renderTaskCard } from '../DOM/taskDOM';
+import { projectList, Task } from '../modules/constructor';
+import { updateLocalStorage } from '../modules/localStorage';
+import { BulmaModal } from '../modules/bulmaModal';
+
+export const iterateTasks = () => {
+  const list = document.getElementById('list');
+  list.innerHTML = '';
+  const projectID = document.getElementById('listProject').value;
```

Comment:
By supposing that file is the controller, please no DOM access/modification here